msu/systems/keybinds/abstract_keybind.nut

```diff
@@ -33,9 +33,15 @@
 		return this.Description;
 	}
 
-	function getKeyCombinations()
+	function getKeyCombinations( _capitalizeFirst = false )
 	{
 		if (this.KeyCombinations.len() == 0) return "";
+
+		if (_capitalizeFirst)
+		{
+			return this.KeyCombinations.reduce(@(_a, _b) ::MSU.String.capitalizeFirst(_a) + "/" + ::MSU.String.capitalizeFirst(_b));
+		}
+
 		return this.KeyCombinations.reduce(@(_a, _b) _a + "/" + _b);
 	}
```

Comment:
I'm not a fan of this being a parameter, I'd even prefer it just being another function that does the same thing but with your changes, `getCapitalizedKeyCombinations` or something. At least then we can easily refactor things if necessary.

Reply:
Ok I will change it to a separate function.